src/layouts/header.scss

```diff
@@ -226,12 +227,29 @@ $lg: "screen and (max-width : 1024px)";
     & > .languageSelect {
       margin-left: 0;
       margin-bottom: 15px;
+      & > .button {
+        width: 100%;
+      }
       & > .list {
         margin-top: 15px;
         position: static;
+        border: none;
+        text-align: center;
         &:not(.is-shown) {
           display: none;
         }
+        & > strong {
+          &:nth-of-type(n+2) {
+            display: block;
+            margin-top: 8px;
+          }
+        }
+        & > .button,
+        & > .link {
+          margin: 0;
+          width: 100%;
+          text-align: center;
```

Comment:
Now that you don't revert all my changes anymore, this conflicts with the left padding that [I set in the previous PR](https://github.com/cilium/ebpf.io/pull/127/commits/5b72880dd248b2145940bdf3847dab6b1f633d71#diff-6e79bb8c92a419f34f0750c31c0707026c20f2f726d7bf09f6b1fff587f46cccR204). You probably want to reset the left padding here to 6px instead of 40px set generically outside of `@media #{$tablet}`.

Current preview shows:
![preview](https://user-images.githubusercontent.com/17001771/122616570-84f66c00-d082-11eb-9520-7aef1f4e8145.png)